### PR TITLE
Screen-designs

### PR DESCRIPTION
**Pull request by lomesshh**
- In this PR I've added :
-- Routing using React-Router-V6
-- Added all the required npm packages
-- homepage design
-- category page design
-- Login and Signup page design
-- Rules, Questions, Result, Leaderboard page design
-- Profile Design

### CODE REVIEW

File: src/frontend/firebase/firebase.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
+import { getFirestore } from 'firebase/firestore';
+import { getAuth } from "firebase/auth";
+// TODO: Add SDKs for Firebase products that you want to use
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+// Your web app's Firebase configuration
+const firebaseConfig = {
```
Comment (shrey27) [inline src/frontend/firebase/firebase.tsx:9]: save the key values in env file

**Review by shrey27 — APPROVED** [review]
Code looks good
PR is ready to merge

File: src/frontend/firebase/firebase.tsx (hunk 1 above)
Comment (lomesshh) [reply src/frontend/firebase/firebase.tsx:9]: done

**Review by romabulani — APPROVED** [review]
- Clean and modular code
- Good to merge
Suggestion: It is a good practice not to push .env file in github, you can declare the variables in netlify when you host it in later PR

**Review by Aman-Jain1996 — APPROVED** [review]
Code looks good and functionality works fine
PR is ready to merge

**Comment by lomesshh** [comment]
> * Clean and modular code
> * Good to merge
>   Suggestion: It is a good practice not to push .env file in github, you can declare the variables in netlify when you host it in later PR

Yup ill do the same before final hosting, Thank you !

**Pull request merged**